From: Sophie Brun <sophie@offensive-security.com>
Date: Mon, 19 Aug 2019 09:08:03 +0200
Subject: Update include to match with Debian package

Last-Update: 2019-02-05

headers of ndpi are in /usr/include/ndpi/ndpi_api.h in Debian package,
not in /usr/include/libndpi/ndpi_api.h 
Last-Update: 2019-02-05
---
 dissectors/tcp_ca/tcp_analysis.c  | 2 +-
 dissectors/tcp_ca/tcp_analysis.h  | 2 +-
 dissectors/tcp_grbg/tcp_garbage.c | 6 +++---
 dissectors/udp_ca/udp_analysis.c  | 4 ++--
 dissectors/udp_grbg/udp_garbage.c | 6 +++---
 5 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/dissectors/tcp_ca/tcp_analysis.c b/dissectors/tcp_ca/tcp_analysis.c
index 510a83e..de0d453 100644
--- a/dissectors/tcp_ca/tcp_analysis.c
+++ b/dissectors/tcp_ca/tcp_analysis.c
@@ -48,7 +48,7 @@
 #include "png.h"
 
 /* nDPI library */
-#include <libndpi/ndpi_main.h>
+#include <ndpi/ndpi_main.h>
 
 
 #define CA_CHECK_LOST     0           /* check lost data */
diff --git a/dissectors/tcp_ca/tcp_analysis.h b/dissectors/tcp_ca/tcp_analysis.h
index 445aa60..c77d392 100644
--- a/dissectors/tcp_ca/tcp_analysis.h
+++ b/dissectors/tcp_ca/tcp_analysis.h
@@ -29,7 +29,7 @@
 #define __TCP_ANALYSIS_H__
 
 #include "pei.h"
-#include <libndpi/ndpi_api.h>
+#include <ndpi/ndpi_api.h>
 
 /* path & buffer size */
 #define TCP_CA_FILENAME_PATH_SIZE      512
diff --git a/dissectors/tcp_grbg/tcp_garbage.c b/dissectors/tcp_grbg/tcp_garbage.c
index ad00a2f..4797a7d 100644
--- a/dissectors/tcp_grbg/tcp_garbage.c
+++ b/dissectors/tcp_grbg/tcp_garbage.c
@@ -48,9 +48,9 @@
 #include "pcap_gfile.h"
 
 /* nDPI library */
-#include <libndpi/ndpi_main.h>
-#include <libndpi/ndpi_api.h>
-#include <libndpi/ndpi_typedefs.h>
+#include <ndpi/ndpi_main.h>
+#include <ndpi/ndpi_api.h>
+#include <ndpi/ndpi_typedefs.h>
 
 
 #define GRB_FILE           0           /* to put (or not) data in to a pcap file */
diff --git a/dissectors/udp_ca/udp_analysis.c b/dissectors/udp_ca/udp_analysis.c
index 66b3253..d98f113 100644
--- a/dissectors/udp_ca/udp_analysis.c
+++ b/dissectors/udp_ca/udp_analysis.c
@@ -46,8 +46,8 @@
 #include "geoiploc.h"
 
 /* nDPI library */
-#include <libndpi/ndpi_main.h>
-#include <libndpi/ndpi_api.h>
+#include <ndpi/ndpi_main.h>
+#include <ndpi/ndpi_api.h>
 
 #define UDP_CA_TMP_DIR    "udp_ca"
 #define NDPI_TICK_RES      1000        /* Hz */
diff --git a/dissectors/udp_grbg/udp_garbage.c b/dissectors/udp_grbg/udp_garbage.c
index e26fdc1..1acf1cc 100644
--- a/dissectors/udp_grbg/udp_garbage.c
+++ b/dissectors/udp_grbg/udp_garbage.c
@@ -47,9 +47,9 @@
 #include "pcap_gfile.h"
 
 /* nDPI library */
-#include <libndpi/ndpi_main.h>
-#include <libndpi/ndpi_api.h>
-#include <libndpi/ndpi_typedefs.h>
+#include <ndpi/ndpi_main.h>
+#include <ndpi/ndpi_api.h>
+#include <ndpi/ndpi_typedefs.h>
 
 #define GRB_FILE           0  /* to put (or not) data in to a file */
 #define UDP_GRB_TMP_DIR    "udp_grb"
